x-pack/libbeat/common/aws/credentials_test.go

```diff
@@ -55,6 +55,7 @@ func TestEnrichAWSConfigWithEndpoint(t *testing.T) {
 			awssdk.Config{},
 			awssdk.Config{
 				EndpointResolver: awssdk.ResolveWithEndpointURL("https://cloudwatch.us-west-1.amazonaws.com"),
+				Region:           "us-west-1",
```

Comment:
Does the other testcase validate that the region defaults to `us-east-1` if it's empty?

Reply:
No, it is unrelated. Defaulting to `us-east-1` is temporary for AWS input while authenticating. The `ResolveWithEndpointURL` functions are used for finding the resources to read from.